builtin_command.go

```diff
@@ -5,10 +5,12 @@ import (
 )
 
 type builtinCommand struct {
+	entrypoint *Entrypoint
 	parent     Module
 	definition *EmbeddedCommand
 }
 
+func (c *builtinCommand) Entrypoint() *Entrypoint  { return c.entrypoint }
```

Comment:
Are we anticipating a consumer will need access to the internal object pointer from accessing it this way and that we won't be able to pass it explicitly to other consumers?

Reply:
Similar exposure in other areas